lib/internal/event_target.js

```diff
@@ -86,7 +86,9 @@ class Event {
     if (arguments.length === 0)
       throw new ERR_MISSING_ARGS('type');
     if (options !== null)
-      validateObject(options, 'options');
+      validateObject(options, 'options', {
```

Comment:
For performance reasons, It's likely better to introduce a separate validator function for this.